lib/RTCSession.js

```diff
@@ -1420,6 +1420,20 @@ module.exports = class RTCSession extends EventEmitter
   {
     logger.debug('sendRequest()');
 
+    if (!this._dialog)
```

Comment:
Let's reorder this and try first to send it over `this._dialog` and if not present use an early dialog if present.

Reply:
Since our main intention here is to use the stablished dialog.